src/plugins/intel_cpu/src/nodes/gather_tree.cpp

```diff
@@ -186,7 +186,7 @@ void GatherTree::GatherTreeExecutor::exec(const MemoryPtr& stepIdxMemPtr,
     parallel_for2d(batchSize, beamWidth, [&](size_t batch, size_t beam) {
         int32_t maxSequenceInBeam = std::min<int32_t>(maxTime, static_cast<int32_t>(maxSeqLen[batch]));
         if (maxSequenceInBeam > 0) {
-            int32_t time;
+            int32_t time = 0;
```

Comment:
int32_t time = maxTime - 1 and initialization in a 'for' loop can be removed

Reply:
Done